packages/playground/remote/src/lib/playground-mu-plugin/0-playground.php

```diff
@@ -91,6 +91,32 @@ function networking_disabled() {
 	<?php
 });
 
+/**
+ * Adds target="_blank" to external links when clicked to open them in a new tab.
+ * This prevents users from loading non-Playground pages inside the Playground iframe.
+ */
+function playground_add_target_blank_to_external_links() {
+	// Only run on frontend and admin pages, not during AJAX requests or CLI
+	if (empty($_SERVER['REQUEST_URI']) || wp_doing_ajax() || wp_doing_cron()) {
+		return;
+	}
+	?>
+	<script>
+		document.addEventListener('click', e => {
+			const a = e.target.closest('a[href]');
+			if (!a) return;                                 // not a link
+			if (new URL(a.href, location).origin !== location.origin) {
+				a.target = '_blank';                        // open in new tab/window
+				a.rel ||= 'noopener';                       // cuts the opener reference
```

Comment:
Should `noopener` be ensured? This only adds it if not present or empty, right?

Reply:
I removed noopener entirely, thank you